tests/suites/1_stateful/09_http_handler/09_0001_json_response.result

```diff
@@ -0,0 +1,3 @@
+{"code":1025,"message":"Unknown table 't1'"}
```

Comment:
What's the different between `code` and `error.code`?

Reply:
they are all .error.code

the first contain fields like .queryid so we only compare .error field